cluster/schema/schema.go

```diff
@@ -113,11 +113,6 @@ func (s *schema) ClassEqual(name string) string {
 }
 
 func (s *schema) unsafeClassEqual(name string) string {
-	for alias := range s.aliases {
```

Comment:
Removed this check, because we don't need it. If you look at the usage, this method is purely to check if given `name` exists in `classes` map.

We use this also to check when someone trying to create alias with name that already exists as collection name. We do have[ tests to lock that behavior already.](https://github.com/weaviate/weaviate/blob/kavirajk%2Fmake-alias-name-case-insensitive/cluster/schema/schema_test.go#L384-L390)